plugins/modules/pnp_intent.py

```diff
@@ -838,23 +874,30 @@ class instance for further use.
             op_modifies=True,
             params=provisioned_count_params,
         )
+        self.log("Devices which are provisioned response from the API 'get device count' is {0}".format(str(prov_dev_response)), "DEBUG")
+
         plan_dev_response = self.dnac_apply['exec'](
             family="device_onboarding_pnp",
             function='get_device_count',
             op_modifies=True,
             params=planned_count_params,
         )
+        self.log("Devices which are in planned state response from the API 'get device count' is {0}".format(str(plan_dev_response)), "DEBUG")
+
         dev_details_response = self.dnac_apply['exec'](
             family="device_onboarding_pnp",
             function="get_device_by_id",
             params={"id": self.have["device_id"]}
         )
+        self.log("Device's details from the API 'get device by id' is {0}".format(str(dev_details_response)), "DEBUG")
```

Comment:
self.log("Response from 'get_device_by_id' API for device details: {0}".format(str(dev_details_response)), "DEBUG")

Reply:
Resolved